mmv1/templates/terraform/custom_expand/runtime_update_policy_enable.go.erb

```diff
@@ -0,0 +1,15 @@
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d tpgresource.TerraformResourceData, config *transport_tpg.Config) (interface{}, error) {
+  l := v.([]interface{})
+  if len(l) == 0 || l[0] == nil {
+    return nil, nil
+  }
+  raw := l[0]
+  field := raw.(map[string]interface{})
+
+  if !field["enable_policy"].(bool) {
```

Comment:
enable_policy doesn't seem to be defined. Is that intentional?

Reply:
oop. good point. these are from an earlier revision. Removing them as they are not used.